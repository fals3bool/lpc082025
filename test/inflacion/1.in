10 3
11 -3 -5 2 20 -10 30 4 -100 22
