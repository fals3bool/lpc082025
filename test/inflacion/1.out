5 7
40
3
